src/bokeh/models/callbacks.py

```diff
@@ -140,15 +141,23 @@ def __init__(self, *args, **kwargs) -> None:
 
     .. code-block: javascript
 
-        export default function(args, obj, data) {
+        export default function(args, obj, data, context) {
             // program logic
         }
 
     where ``args`` is a key-value mapping of user-provided parameters, ``obj``
-    refers to the object that triggered the callback, and ``data`` is a key-value
-    mapping of optional parameters provided by the caller.
+    refers to the object that triggered the callback, ``data`` is a key-value
+    mapping of optional parameters provided by the caller, and ``context`` is
+    additional context.
 
     This function can be an asynchronous function (``async function``).
+
+    The addtional context is composed of the following members:
```

Comment:
suggest moving this block about context immediately after the `and context is additional context" and move the sentenct about async after that, at the end.